lib/account/account-manager.js

```diff
@@ -1,36 +1,80 @@
 "use strict";
-/**
- * Manages the business logic of account controls.
- */
 
 /**
- * The object sent in which has information needed to validate the user
+ * The object sent container information needed to validate the user
  * has properly set up their two factor authentication and has generated
- * a password for use to save.
+ * a password for use.
  *
  * @typedef {Account~accountinfo}
- * @param {string} accountId this is the unhashed account id sent to client
- * @param {string} currentMfa
- * @param {string} previousMfa
- * @param {string} password
+ * @property {string} accountId this is the unhashed account id sent to client
```

Comment:
You don't ever need to say "this is" for the descriptions.  You really don't need "the" either.